RELEASE.md

```diff
@@ -12,24 +12,24 @@ Bug fixes are expected to be worked on and applied to `master`.  If the fix is n
 
 ## Release Request Process
 
-The KUDO Project is released on an as-needed basis. The process is as follows:
+The KUDO Project is aiming to do monthly minor releases but the period could be even shorter if necessary. The process is as follows:
 
-1. An issue is proposing a new release with a changelog since the last release
-1. All [OWNERS](OWNERS) must LGTM this release
-1. Release process below is followed
-1. The release issue is closed
+1. Every release should have an appointed release manager (RM)
+1. RM is responsible for following the process below
+1. RM should announce the release in the public [#kudo slack channel](https://kubernetes.slack.com/messages/kudo/) at least two days prior to the date
```

Comment:
Thanks for adding more details here! But shouldn't the release get announced _after_ it has been released?

Reply:
maybe I need to change the wording? What I meant was that we usually should let the community and other members of the team know that we'll be cutting a release :)

Reply:
Oh, that's what's meant here. No, the wording is good, I'm just not used to projects announcing releases beforehand :D